content/docs/05-ai-sdk-ui/21-error-handling.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Error Handling
+description: Learn how to handle errors in the AI SDK UI
+---
+
+# Error Handling
+
+Errors can be handled by passing an [`onError`](/docs/reference/ai-sdk-ui/use-chat#on-error) callback function as an option to the [`useChat`](/docs/reference/ai-sdk-ui/use-chat), [`useCompletion`](/docs/reference/ai-sdk-ui/use-completion) or [`useAssistant`](/docs/reference/ai-sdk-ui/use-assistant) hooks.
+
+Each AI SDK UI hook also returns an [error](/docs/reference/ai-sdk-ui/use-chat#error) object that you can use to render the error in your UI.
```

Comment:
The example should show both, i.e. also how you display an error (if there is one), since it's fairly common

Reply:
they could be separated into 2 subsection ("onError callback", "error helper") - they serve 2 different purposes (logging/handling, error rendering/informing the user)